text/0012-cva.md

```diff
@@ -0,0 +1,161 @@
+- that the number of active validators is bounded by a protocol imposed ceiling;
+- that byzantine validators are present, even at genesis;
+- that whilst active markets exist for the network token, they are assumed to be mostly unregulated;
+- that the network's crypto-economic security model is predicated upon so-called Proof-of-Stake (POS);
+- that via a smart contract, the network exposes an auction mechanism permitting prospective new validators' to signal an intent to participate;
+- that the network's governance mechanisms revolve around off-chain social consensus;
+- that it is in the interests of all network participants to adopt a pro-active approach to network-wide risk management.
+- that a validator may onboard as a result of attracting sufficient delegated stake weight - I.e. the validator's associated risk is based upon reputation rather than capital. 
+
+## Motivation
+
+[motivation]: #motivation
+
+It is imperative that the Casper network validator set can safely evolve over time.  It is inconceivable that the network does not support such a feature at genesis, i.e. it is a non-negotiable requirement of a minimal viable main-net.  Thus the CEP motivation is crystal clear, it is a necessary step towards a **demonstrably** healthy network.  Healthy from an operational, crypto-economic & reputational perspective.  
+
+A subset of the contextual use cases this CEP touches upon are:
+
+1.  The reputation of the network must be such that (new) validator's are willing to continue to assume non-trivial capital risk in order to participate;
+
+2.  The ratio of staked to non-staked token supply has game theoretic dynamics & hence real consequences;  
+
+3.  The inability of the token holder base to secure an ambient ROI (via inflationary rewards) results in a slow drift towards centralisation;
+
+## Guide-level explanation
+
+[guide-level-explanation]: #guide-level-explanation
+
+### 1. Establish a Casper Validator Association (CVA)
+
+Running a validator node entails significant risk.  Such risk revolves around the following elements:
+
+- Existential issues with the node software - e.g. a hitherto undiagnosed fault;
+- Operational issues with the node deployment - e.g. a concerted DDOS attack;
+- Jurisdictional issues associated with the business context - e.g. new compliance regulations.
+- Reputational issues within a social consensus context - e.g. cartel formation.
+
+A rational risk mitigation strategy dictates that validators collaborate.  Timely information sharing over well defined communication channels is the backbone of such collaboration. 
+Validators operating in isolation are disempowered as compared to collaborative ad-hoc groups of validators.  However ad-hoc groups tend to coalesce into political cliques focussing upon narrow tactical objectives at the expense of coherent strategic objectives.  This strategic incoherence is to the detriment of the network's long-term viability.
```

Comment:
The distinction that should have been made here is not between tactical and strategic objectives, but between those incentives that drive rational validators to weaken the platform (by over-centralizing, for example) and those that do not. That should clarify the nature of the problem we're facing with organizing validators into cliques.